src/core/core-adapter.js

```diff
@@ -587,6 +587,19 @@ export default class CoreAdapter {
     this.filterRegistry.clearLocationRadiusFilterNode();
   }
 
+  /**
+   * Gets the location object needed for answers-core
+   * 
+   * @returns {LatLong|undefined} from answers-core
+   */
+  _getLocationPayload () {
+    const geolocation = this.storage.get(StorageKeys.GEOLOCATION);
+    return geolocation && {
+      latitude: geolocation.lat,
+      longitude: geolocation.lng,
```

Comment:
nit: I don't think we do trailing commas in the SDK (though I am not opposed to that in the future)

Reply:
ah, I had the linter disabled temporarily and missed this